Meta/run.py

```diff
@@ -0,0 +1,978 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2018-2023, the SerenityOS developers.
+# Copyright (c) 2023, kleines Filmröllchen <filmroellchen@serenityos.org>
+#
+# SPDX-License-Identifier: BSD-2-Clause
+
+from __future__ import annotations
```

Comment:
`__future__`? What version of python3 are we using that doesn't have these?

Reply:
"Advanced" annotations are an opt-in feature even as of current 3.11/3.12. It's standard practice in fully type-checked files such as this one (yes, I ran mypy over the script).